vector/src/main/res/values/strings.xml

```diff
@@ -959,6 +959,7 @@
     <string name="room_delete_unsent_messages">Delete unsent messages</string>
     <string name="room_message_file_not_found">File not found</string>
     <string name="room_do_not_have_permission_to_post">You do not have permission to post to this room.</string>
+    <string name="room_unsupported_e2e_algorithm">Encryption has been misconfigured. You can\'t send messages, please contact an admin to restore the room to a valid state.</string>
```

Comment:
Do we show this message even when the user IS an admin?
If yes, we could probably add a way to open the settings in this case.

Reply:
added

Reply:
You added a way to open the setting, but maybe also update the message for admins to:
`Encryption has been misconfigured. You can\'t send messages. Click to open the settings to restore the room to a valid state.`

Reply:
Done